openid-provider-commands-1_0.md

```diff
@@ -999,6 +944,119 @@ Streaming Response Events:
 - `account-state` events MUST include: `sub` (REQUIRED), `account_state` (value `active`)
 - A final `command-complete` event MUST include: `total_accounts` (REQUIRED)
 
+
+# Notifications
+
+
+Notifications enable an RP to notify an OP of a change in the RP state or the completion of an asynchronous command. Notifications are authenticated using a Notification Token issued and signed by the RP.
+
+## Notification Token
+
+A Notification Token is a JSON Web Token (JWT) signed by the RP. The OP validates the signature using the RP's published keys (e.g., via the `jwks_uri` value that the RP provided in the Metadata Response). To avoid cross-JWT confusion, Notification Tokens MUST include a `typ` JWS header of `notification+jwt`.
```

Comment:
You will eventually need to add an IANA media type registration for application/notification+jwt.